src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -112,8 +119,11 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
       issueReply using updatedData
   }
 
+  var isDone = false
```

Comment:
The `FSM[StateClass, DataClass]` has stateName and stateData for getting the current state.